smart-client-jersey/src/main/java/com/emc/rest/smart/jersey/SmartClientFactory.java

```diff
@@ -43,6 +46,10 @@ public final class SmartClientFactory {
     public static final int MAX_CONNECTIONS_DEFAULT = 999;
     public static final int MAX_CONNECTIONS_PER_HOST_DEFAULT = 999;
 
+    public static final String MAX_CONNECTION_IDLE_TIME = "com.emc.rest.smart.apacheMaxConnectionIdleTime";
```

Comment:
instead of a custom property, this should be a named property in `SmartConfig` (`SmartConfig.maxConnectionIdleTime`).  It would only apply when using the Apache client (this should be in the javadoc of the getter/setter), and the default value would be 0 (off).